Remix recipe not running

## Description

The Puck Remix recipe does not run with either `npm run dev` or `yarn dev`; both commands throw a build-time error. This is because the `tiny-invariant` library is not installed as a dependency by default.

## Environment

- Puck version: 0.18.2

## Steps to reproduce

1. Create a new Puck project using the Remix recipe.
2. Open the project folder.
3. Run the project in development mode.

## What happens

A build-time error occurs:
```bash
X [ERROR] Build failed with 1 error:
app/routes/edit.tsx:11:22: ERROR: Could not resolve "tiny-invariant" [plugin css-bundle-plugin]
```

__You can fix this by manually installing `tiny-invariant`:__:

```bash
npm install tiny-invariant
```

## What I expect to happen

The recipe should work out of the box without any errors.

--- a/packages/create-puck-app/templates/remix/package.json.hbs
+++ b/packages/create-puck-app/templates/remix/package.json.hbs
@@ -18,7 +18,8 @@
     "@remix-run/serve": "^2.2.0",
     "isbot": "^3.6.8",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "tiny-invariant": "^1.3.3"
   },
   "devDependencies": {
     "@remix-run/dev": "^2.2.0",

--- a/recipes/remix/package.json
+++ b/recipes/remix/package.json
@@ -18,7 +18,8 @@
     "@remix-run/serve": "^2.2.0",
     "isbot": "^3.6.8",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "tiny-invariant": "^1.3.3"
   },
   "devDependencies": {
     "@remix-run/dev": "^2.2.0",

